### ISSUE TITLE
NewsfeedNewsfeedItem not contains most of availiable fields

### ISSUE THREAD

**Issue by magicxor** [khrabrovart/Citrina#3]
According to https://vk.com/dev/newsfeed.get it should be like

```
    public class NewsfeedNewsfeedItem
    {
        public string Type { get; set; }
        public int? SourceId { get; set; }
        public DateTime? Date { get; set; }
        public int? PostId { get; set; }
        public string PostType { get; set; }
        public int? CopyOwnerId { get; set; }
        public int? CopyPostId { get; set; }
        public string Text { get; set; }
        public bool? CanEdit { get; set; }
        // ...
    }
```

**Comment by khrabrovart** [comment]
Fixed. Thank you!

**Comment by magicxor** [comment]
@khrabrovart I checked it today, unfortunately it still hasn't some fields such as Date, SourceId, NextFrom which are very important for my app. It could be amazing if you add this fields.

**Comment by khrabrovart** [comment]
@magicxor Thanks for pointing this out. 
Updated the NewsfeedNewsfeedItem and NewsfeedGetResponse classes according to the actual VK response. Published [Citrina 3.0.1](https://www.nuget.org/packages/Citrina/)
